### ISSUE TITLE
Add Fathom analytics to our website

### ISSUE THREAD

**Issue by ReinhardSchmidt** [hoprnet/hoprnet-org#147]
# Page
Entire website (head component)

# Current behavior
The site is missing the fathom tracker, a privacy-friendly tracker alternative to Google. We need to add it from our last site to this one, we forgot to port this.

# Possible solution
- [ ] Add the following lines to the `App.js` of our website (see [our previous website](https://github.com/hoprnet/website-hoprnet-org/blob/9b0f4b76ef30decbf8a0febf2c11efbf7091b95b/src/App.js#L33-L36)) so Fathom starts tracking again.

```
// add fathom analytics
    const script = insertScript('https://panther.hoprnet.org/script.js')
    script.setAttribute('site', 'ZXTSKLDN')
    script.setAttribute('spa', 'auto')
```

**Comment by ruddygasol94** [comment]
@jjperezaguinaga  Do we already have the logo?

Thanks!

**Comment by jjperezaguinaga** [comment]
Hey @ruddygasol94 , unfortunately this ticket is a bit misleading. Let me rephrase it a bit.

**Comment by jjperezaguinaga** [comment]
@ruddygasol94 Ready for you!

**Comment by ruddygasol94** [comment]
Excellent, thanks!
